builtin/providers/google/resource_compute_instance_template.go

```diff
@@ -134,7 +143,6 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 			"automatic_restart": &schema.Schema{
 				Type:       schema.TypeBool,
 				Optional:   true,
-				Default:    true,
```

Comment:
I'm not sure a state_migration will be able to fix the issue. Previously a config without auto_restart or scheduler set will create a default scheduler with auto_restart set to true (which seems like odd behavior to me, especcially for a deprecated field). Taking away this default will make configs such as this destroy the scheduler. To keep the scheduler it seems like I would need to migrate configs rather than states somehow. Thoughts on this?